pyfar/constants/__init__.py

```diff
@@ -8,3 +9,95 @@
 __all__ = [
     'speed_of_sound_simple',
 ]
+
+
+standard_atmosphere_pressure: Final[float] = 101325.0
+"""
+Standard atmosphere pressure in Pa as defined in [#]_.
+
+.. math::
+    P_{atm} = 101325 Pa
```

Comment:
Units and non mathematical symbols should not be italic and there should be spaces between numbers and units. Does this
```python
.. math::
    P_\text{atm} = 101325 \, \text{Pa}
```
Work in sphinx? Otherwise a compromise could be
```python
:math:`P_{atm}=101325` Pa
```

This pertains to almost all units in the pull